simcon_project/static/js/disable_back_button.js

```diff
@@ -0,0 +1,7 @@
+window.onunload = function () { null };
+function preventBack() {
+    window.history.forward();
```

Comment:
instead of just going forward cant you do what websites do that just take u back to main? so student view or researcher view... that trained me to not use the back button

Reply:
Haha that's a pretty "tough love" alternative, but that could be what we want... I guess I tried to be a little more charitable.